src/components/Ratings/Rating.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect, FC } from "react";
+import StarIcon from "./StarIcon";
+import "./_Star.scss";
+
+export interface IRatingProps {
+    /**set true or false*/
+    disableHandler?: boolean;
+    /**set "sm" | "md" | "lg" */
+    starSize?: string;
+    /**set custom classname by sending in a prop*/
+    classNames?: string;
+    /**set star count by using array [1,2,3,4,5]*/
+    starCounts?: number[];
+    /**set customized default value of star. Example: 3*/
+    customRatingValue?: number;
+    /**set true or false to activate label on hover*/
+    hoverLabel?: boolean;
+    /**set custom icon size by sending prop in. Example: "75px" */
+    customSize?: string;
+    /**set label title by sending in prop. Example: "TESTING LABEL"*/
+    labelTitle?: string;
+    /**send addon-action from storybook as a prop */
+    onChange?: any
```

Comment:
The type you set here is very accurate, also the comments did help people understand your code better. However, based on the rest of codes, I believe that your onChange here is a callback function, so it would be better if we use () => any instead of any. And of course, that's just my opinion.

Reply:
thank you i will make that change.